istio/tests/test_istio_v1.py

```diff
@@ -36,7 +36,7 @@ def test_legacy_proxy_mesh(aggregator, dd_run_check):
     for metric in common.LEGACY_MESH_METRICS + common.MESH_MERICS_1_5:
         aggregator.assert_metric(metric)
 
-    _assert_tags_excluded(aggregator, [])
+    _assert_tags_excluded(aggregator, [], True)
```

Comment:
Could you add the argument name here for readability plz? I dunno what it is and `True` on its own doesn't say what's happening, what we're enabling.

Same goes for line 61 btw